e2e_test/tests/load_async.spec.ts

```diff
@@ -2,7 +2,7 @@ import { test } from '@playwright/test';
 import { Server } from 'http';
 import axios from 'axios';
 import path from 'path';
-import fs from 'fs/promises';
+import { promises as fs } from 'fs';
```

Comment:
Previous syntax was fine, but you need to have the correct node version (use `nvm use` in the main directory).

Reply:
done